apache-spark/notebook/Image_Classification_Spark.ipynb

```diff
@@ -82,38 +89,38 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "  // Translator: a class used to do preprocessing and post processing\n",
-    "  class MyTranslator extends Translator[Row, Classifications] {\n",
-    "\n",
-    "    private var classes: java.util.List[String] = new util.ArrayList[String]()\n",
-    "    private val pipeline: Pipeline = new Pipeline()\n",
-    "      .add(new Resize(224, 224))\n",
-    "      .add(new ToTensor())\n",
-    "\n",
-    "    override def prepare(manager: NDManager, model: Model): Unit = {\n",
-    "        classes = Utils.readLines(model.getArtifact(\"synset.txt\").openStream())\n",
-    "      }\n",
-    "\n",
-    "    override def processInput(ctx: TranslatorContext, row: Row): NDList = {\n",
-    "\n",
-    "      val height = ImageSchema.getHeight(row)\n",
-    "      val width = ImageSchema.getWidth(row)\n",
-    "      val channel = ImageSchema.getNChannels(row)\n",
-    "      var image = ctx.getNDManager.create(ImageSchema.getData(row), new Shape(height, width, channel)).toType(DataType.UINT8, true)\n",
-    "      // BGR to RGB\n",
-    "      image = image.flip(2)\n",
-    "      pipeline.transform(new NDList(image))\n",
-    "    }\n",
+    "// Translator: a class used to do preprocessing and post processing\n",
```

Comment:
Should we use spark extension in the jupyter notebook?

Reply:
OK